src/services/groups.service.ts

```diff
@@ -0,0 +1,96 @@
+import { PrismaClient, Group } from '@prisma/client';
+import { CreateGroupDto } from '@dtos/group.dto';
+import { HttpException } from '@exceptions/HttpException';
+import { isEmpty } from '@utils/util';
+
+class GroupService {
+  public group = new PrismaClient().group;
+
+  public async findAllGroup(): Promise<Group[]> {
```

Comment:
The method name findAllGroup uses a singular form, which is inconsistent with the naming convention for retrieving collections.
```suggestion
  public async findAllGroups(): Promise<Group[]> {
```
Additionally, returning all records at once could lead to performance issues with large datasets. Consider implementing pagination and filtering support to improve scalability.

Lastly, I don't see any sorting logic applied to the results. It's recommended to return data in a consistent sort order for predictability and a better user experience.